docs/AboutUs.md

```diff
@@ -26,17 +26,17 @@ You can reach us at the email `what email should we put here?`
 [[github](http://github.com/johndoe)]
 [[portfolio](team/johndoe.md)]
 
-* Role: Team Lead
-* Responsibilities: UI
+- Role: Team Lead
+- Responsibilities: UI
 
-## Johnny Doe
+### Ethan Chen Ee Shuen
```

Comment:
I think we are using h2 headers for names?

Reply:
Yeah, I've changed it. Thanks for spotting the mistake.